.env

```diff
@@ -1,16 +1,17 @@
 # This file is commited. Do not store secrets here
```

Comment:
Let's use the following scheme for _all_ the env variables: `REACT_APP_${ENV}_${NAME}`, so for example:
- `REACT_APP_PRODUCTION_QUERY_NODE_URL`
- `REACT_APP_STAGING_QUERY_NODE_URL`